### DISCUSSION TITLE
Install issue on EndeavourOS

### DISCUSSION THREAD

**Discussion by changeisalchemy** [estarq/ramdisk#2] [Q&A]
Hi estarq,

Thanks for making RAMdisk. Unfortunately, I can't install it. I get to   sudo python3 install.py  and I get this output:
Thanks for any help you can offer.

Cheers
A

[alpha@alpha1 ramdisk]$ sudo python3 install.py
[sudo] password for alpha: 
:: Synchronising package databases...
 core                            137.5 KiB   121 KiB/s 00:01 [--------------------------------] 100%
 extra                          1559.5 KiB   697 KiB/s 00:02 [--------------------------------] 100%
 community                         6.0 MiB  1557 KiB/s 00:04 [--------------------------------] 100%
 multilib is up to date
 endeavouros                      24.2 KiB  23.0 KiB/s 00:01 [--------------------------------] 100%
error: target not found: python3-gi
error: target not found: python3-gi-cairo
error: target not found: gir1.2-gtk-3.0
warning: adwaita-icon-theme-41.0-1 is up to date -- reinstalling

I ran the command again and got this:

[alpha@alpha1 ramdisk]$ sudo python3 install.py
[sudo] password for alpha: 
:: Synchronising package databases...
 core is up to date
 extra is up to date
 community is up to date
 multilib is up to date
 endeavouros is up to date
error: target not found: python3-gi
error: target not found: python3-gi-cairo
error: target not found: gir1.2-gtk-3.0
warning: adwaita-icon-theme-41.0-1 is up to date -- reinstalling
mv: cannot stat 'data/ramdisk': No such file or directory
mv: cannot stat 'data/ramdisk.desktop': No such file or directory
mv: cannot stat 'data/ramdisk.png': No such file or directory
mv: cannot stat 'data/ramdisk.policy': No such file or directory
mkdir: cannot create directory ‘/usr/share/ramdisk/’: File exists
mv: cannot stat 'data/data.json': No such file or directory

**Comment by estarq** [comment]
Hi,

sorry for the late reply. I've downloaded the latest Endeavour release (EndeavourOS_Atlantis_neo-21_5.iso) and installed RAMDisk without any problems. You shouldn't worry about "target not found" errors, and it's perfectly normal that you got all of those additional errors on the second run, because the files have already been moved (not copied) during the first run, so they're not there anymore. The output of install.py doesn't look good (I will change that later), but RAMDisk should be installed anyway. I can run RAMDisk from the "start menu" (it's in Accessories), by pressing alt+F2 and typing "ramdisk", and by executing "ramdisk" in Terminal. Does any of those methods work for you? If not, what's the output of "ramdisk" in Terminal?
